inception-search-mtas/pom.xml

```diff
@@ -29,7 +29,8 @@
   <dependencies>
     <dependency>
       <groupId>de.tudarmstadt.ukp.clarin.webanno</groupId>
-      <artifactId>webanno-api</artifactId>
+      <artifactId>webanno-project</artifactId>
+      <scope>test</scope>
```

Comment:
Please move this down to the other test dependencies

Reply:
OK.